charts/humio-operator/values.yaml

```diff
@@ -49,3 +49,11 @@ certmanager: true
 defaultHumioCoreImage: ""
 defaultHumioHelperImage: ""
 defaultHumioHelperImageManaged: ""
+
```

Comment:
We don't need these now right?

Reply:
This comment does not really show exactly where the additional lines are. Looking at the complete file I can see that are remaining pdf render server values